Source/WebCore/css/parser/CSSPropertyParserHelpers.h

```diff
@@ -433,24 +320,95 @@ RefPtr<CSSValue> consumeGridTrackList(CSSParserTokenRange&, const CSSParserConte
 RefPtr<CSSValue> consumeGridTemplatesRowsOrColumns(CSSParserTokenRange&, const CSSParserContext&);
 RefPtr<CSSValue> consumeGridTemplateAreas(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeLineBoxContain(CSSParserTokenRange&);
-RefPtr<CSSValue> consumeWebkitLineGrid(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeContainerName(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeWebkitInitialLetter(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeSpeakAs(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeHangingPunctuation(CSSParserTokenRange&);
-RefPtr<CSSValue> consumeWebkitMarqueeIncrement(CSSParserTokenRange&, CSSParserMode);
-RefPtr<CSSValue> consumeWebkitMarqueeRepetition(CSSParserTokenRange&);
-RefPtr<CSSValue> consumeWebkitMarqueeSpeed(CSSParserTokenRange&, CSSParserMode);
 RefPtr<CSSValue> consumeAlt(CSSParserTokenRange&, const CSSParserContext&);
 RefPtr<CSSValue> consumeContain(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeContainIntrinsicSize(CSSParserTokenRange&);
 RefPtr<CSSValue> consumeTextEmphasisPosition(CSSParserTokenRange&);
 #if ENABLE(DARK_MODE_CSS)
 RefPtr<CSSValue> consumeColorScheme(CSSParserTokenRange&);
 #endif
-RefPtr<CSSValue> consumeListStyleType(CSSParserTokenRange&, const CSSParserContext&);
 RefPtr<CSSValue> consumeOffsetRotate(CSSParserTokenRange&, CSSParserMode);
 
+
+// Template and inline implementations are at the bottom of the file for readability.
+
+template<typename... emptyBaseCase> inline bool identMatches(CSSValueID) { return false; }
```

Comment:
Maybe `constexpr` instead of `inline`. Maybe neither because this is a template and so there’s no keyword needed to put it in a header file without violating the multiple translation unit rule and separately there’s also no keyword needed to get the compiler to inline it.